lib/kdl/include/kdl/vector_utils.h

```diff
@@ -0,0 +1,381 @@
+/*
+ Copyright 2010-2019 Kristian Duske
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+ documentation files (the "Software"), to deal in the Software without restriction, including without limitation the
+ rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit
+ persons to whom the Software is furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all copies or substantial portions of the
+ Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+ WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+ OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+
+#ifndef KDL_VECTOR_UTILS_H
+#define KDL_VECTOR_UTILS_H
+
+#include "collection_utils.h"
+
+// Note: all except <cassert> are included by <vector> anyway, so there's no point in splitting this up further
+#include <cassert>
+#include <algorithm> // for std::sort, std::unique, std::find, std::find_if, std::remove, std::remove_if
+#include <iterator> // std::back_inserter
+#include <functional> // for std::less
+#include <type_traits> // for std::less
+#include <vector>
+
+namespace kdl {
```

Comment:
I commented before but I'd suggest a `vec_` or `vec::` prefix on all of these.

Reply:
I agree, I'll go with `vec_` and `map_`.